.devcontainer/devcontainer.json

```diff
@@ -6,7 +6,7 @@
       },
       "ghcr.io/devcontainers/features/aws-cli:1": {},
       "ghcr.io/devcontainers-contrib/features/aws-cdk:2": {
-        "version": "2.173.4"
```

Comment:
Could we find a way to parameterize this vs changing in so many files?

Reply:
I had similar thoughts, the reason why it has not happened - it takes 10 sec to do global replace now. Parameterizing this as well as references in the doc will take some time and debugging. However, agree with the suggestion, it is a good first issue for someone to start exploring the codebase.

Reply:
Created https://github.com/aws-quickstart/cdk-eks-blueprints/issues/1133